api-server-nest/src/utils/get-common-query-terms-for-events.ts

```diff
@@ -0,0 +1,21 @@
+import { Op, WhereOptions } from "sequelize";
+// import {HttpException, HttpStatus} from "@nestjs/common";
+
+// TODO: it would be nice to be able to say tags=-tag to filter on events _without_ tag
+/**
+ * Build query options for events (or current events)
+ * 
+ * @param verified true for only verified events, false for only unverified events, null for both
+ * @param tags tag or array of tags to filter by (when using an array, only events with ALL tags are returned)
```

Comment:
FYI, one of the nice things about typescript is you can skip this kind of documentation.

I'd argue the function name and type decorations speak for them selves; though you could perhaps change the name to buildQueryOptionsForEventServiceCalls? I'm fine with it either way and definitely, not insisting you delete the comments, but I've been reading Clean Code and they make the case that comments outside of public apis just get in the way and become stale and are indicators that you need better function names or type definitions.